paper.tex

```diff
@@ -254,8 +254,7 @@
+This paper presents the release of EinsteinPy version 0.2, a community-developed Python package which pertains to gravitational and relativistic astrophysics. Python, a free, easy to use high-level programming language has conquered today’s regime of programming and has seen a huge expansion in recent times thereby extending its applicability in the domains of Astrophysics and in general physics. Many great frameworks on Python have substantiated a very high level of abstraction over the intricacies of complex algorithms and thus provide an easy to use interface and pleasing user experience. One such example is Keras - a framework for deep learning which has made the complex area much easier that even a neo-programmer can  train a neural network classifier. This example really demonstrates the power of abstraction which is achievable in Python. EinsteinPy brings forth its essence in a similar way and was developed keeping in mind the state of a theoretical gravitational physicist with a little or no background in computer programming and trying to work in the field of numerical relativity or trying to use simulations in their research.EinsteinPy aims to foster the development and functionality to the astronomical community with currently facilitating the simulation of time-like and null geodesics and calculation of trajectories in different background geometries some of which are Schwarzschild, Kerr and KerrNewmann along with coordinate inter-conversion pipeline. It has a partially developed pipeline for plotting and visualization with dependencies on libraries like plotly, matplotlib etc. One of the unique features of EinsteinPy is sufficiently developed symbolic tensor manipulation utilites which can build a strong foundation by deploying as a tool for self-learning in the area of tensor algebra. Currently, EinsteinPy also renders few utility functions for hypersurface embedding of Schwarzschild spacetime which further will be extended to model gravitational lensing simulation. The current version of the library aids in the path of inherent challenges of substantial mathematical formalism faced by a novice in general relativity, thereby modifying and remoulding the equations into visualisations.
```

Comment:
> The current version of the library aids in the path of inherent challenges of substantial mathematical formalism faced by a novice in general relativity, thereby modifying and remoulding the equations into visualisations.

Changes are good but Please let the last line remain same as the previous one - from "The current ....." to "....whats really happening".